src/main/java/com/booleanuk/core/Exercise.java

```diff
@@ -56,14 +62,24 @@ public ArrayList<Integer> getFavouriteNumbers() {
          https://www.programiz.com/java-programming/library/arraylist/replaceall
      */
 
+    public ArrayList<Integer> multiply(ArrayList<Integer> list, int number){
+        list.replaceAll(e -> (e*number));
+        return list;
 
+    }
 
     /*
         TODO: 3. Create a method named isEmpty that accepts one parameter:
          - A list of strings
          The method must return a boolean that indicates whether the provided list is empty or not
      */
-
+    public boolean isEmpty(ArrayList<String> list){
+        if (list.size()==0){
+            return true;
+        }else{
+            return false;
+        }
+    }
```

Comment:
Could be simplified using:

`return list.size() == 0;`

or even

`return list.isEmpty();`